APIBlueprintGenerator.js

```diff
@@ -93,6 +93,43 @@
       }
       return path;
     };
+    this.parameter = function(paw_request) {
```

Comment:
This file is actually derived from [`APIBlueprintGenerator.coffee`](https://github.com/apiaryio/Paw-APIBlueprintGenerator/blob/master/APIBlueprintGenerator.coffee) so you will need to make these changes from there instead. You can find instructions to build this file in the README (https://github.com/apiaryio/Paw-APIBlueprintGenerator#development-installation).